### PR TITLE
Update business-model.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Removed extraneous periods, changed hyphens to colons, fixed trailing spaces, and some minor rewording for clarity on points.

### CODE REVIEW

File: company/about-mattermost/business-model.md [hunk 1]
```diff
@@ -12,14 +12,11 @@ The paid enterprise edition of Mattermost is built for **team managers** and **I
...
-**Innovation** - Monthly ship cycles and high velocity innovation rapidly lower the value of forks   
-**Compatibility** - Only genuine versions upgrade smoothly to new releases and security patches    
-**Architecture** - Modular architecture makes need to fork rare  
-**Licensing** - Trademark is registered internationally, forking requires renaming   
-  
-
-
+**Test infrastructure:** We use [SaaS QA](https://www.rainforestqa.com/) which isn't included in the open source version 
+**Alignment:** Developer features are free. Paid features are for managers and IT pros, who are less likely to spend time and effort forking and maintaining an open source project vs. deploying budgets to purchase paid features
+**Core committers:** We work with an active community of core committers to keep the project and business in sync
```
Comment (jasonblais) [inline company/about-mattermost/business-model.md]:
```suggestion
**Core committers:** We work with an active community of [core committers](https://developers.mattermost.com/contribute/getting-started/core-committers/) to keep the project and business in sync
```

File: company/about-mattermost/business-model.md [hunk 2]
```diff
@@ -12,14 +12,11 @@ The paid enterprise edition of Mattermost is built for **team managers** and **I
...
+**Threat intelligence:** Mattermost's RDP means forks will lag in threat intelligence
```
Comment (jasonblais) [inline company/about-mattermost/business-model.md]:
```suggestion
**Threat intelligence:** Mattermost's [responsible disclosure policy](https://mattermost.com/security-vulnerability-report) means forks will lag in threat intelligence
```

**Review by jasonblais — DISMISSED** [review]
Two non-blocking suggestions.

File: company/about-mattermost/business-model.md [hunk 3]
```diff
@@ -12,14 +12,11 @@ The paid enterprise edition of Mattermost is built for **team managers** and **I
...
-**Core committers** - We employ vast majority of core committers to keep project and business in sync   
-**Threat intelligence** - Mattermost's RDP means forks will lag in threat intelligence   
-**Innovation** - Monthly ship cycles and high velocity innovation rapidly lower the value of forks   
-**Compatibility** - Only genuine versions upgrade smoothly to new releases and security patches    
-**Architecture** - Modular architecture makes need to fork rare  
-**Licensing** - Trademark is registered internationally, forking requires renaming   
-  
-
-
+**Test infrastructure:** We use [SaaS QA](https://www.rainforestqa.com/) which isn't included in the open source version
```
Comment (amyblais) [inline company/about-mattermost/business-model.md]: These are not rendering in a list format currently.

**Review by amyblais — APPROVED** [review]

**Pull request merged**